### ISSUE TITLE
transfer learning with pose models

### ISSUE THREAD

**Issue by snachi2s** [ultralytics/ultralytics#6493]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hi @glenn-jocher,
Thanks for your great work!!

I trained "yolov8n" and "yolov8n-pose" models with my custom dataset for object detection and keypoint estimation separately. 

Now, I want to freeze the detect layers in "yolov8n-pose" and instead use the weights from "yolov8n"  and only train the pose head for my custom keypoint detection. Is this possible? If yes, can you help me in achieving that? 

Thanks in advance!!

**Comment by glenn-jocher** [comment]
@snachi2s hello!

First off, thank you for your kind words, and I'm glad to hear about your progress with the YOLOv8 models! 

Yes, it is definitely possible to perform the transfer learning task you've described. For your specific use case, you would need to load your pretrained 'yolov8n' weights and then continue training on the 'yolov8n-pose' model while freezing the detection layers.

To accomplish this, you'll need to follow these steps:
1. Prepare your custom dataset formatted correctly for keypoint detection.
2. Load the 'yolov8n' weights into the 'yolov8n-pose' model.
3. Freeze the detection layers using the freeze layers option during training.
4. Set up the rest of the training pipeline by specifying the keypoint annotations and beginning the training process, focusing on the pose estimation head.

This process will ensure that the knowledge learned from object detection is transferred to the pose estimation task, improving the potential accuracy without learning from scratch.

Please refer to our comprehensive Ultralytics Documentation for detailed instructions on how to set up your training configuration and training process. You can find info on model customization and training in different modes that could be particularly helpful. Good luck with your continued work on keypoint estimation! 🚀

If you find yourself needing specific guidance, you're more than welcome to reach out to us within the YOLOv8 GitHub Discussions. Our team and the community are always there to help and share insights.

Best wishes on your model training!

**Comment by snachi2s** [comment]
Thanks for the reply!! 
Here is the script I used for training, I freeze 19 layers (except the head)
```
model = YOLO('yolov8n-pose.pt')
results = model.train(data='keypoint_config.yaml', epochs=100, batch=2, imgsz=640, pretrained='custom_trained_yolov8n_for_obejct_detection.pt', freeze=19)
```
The reason I want to use my trained model and freezing the layers is to detect objects in the image using my loaded weights rather than using the bounding box information from my keypoint labels file. 

If I freeze the layers as mentioned above, the model still utilizes the bounding box information from the label.txt file rather than from the loaded pretrained model. 
I want to use the bounding boxes from the loaded model for training. What changes do I need to make to achieve that?
